File: 프로그래머스/lv2/12951. JadenCase 문자열 만들기/JadenCase 문자열 만들기.js
function solution(s) {
  let arr = s.split(" ");
  let result = [];
  for (i = 0; i < arr.length; i++) {
    if (arr[i]) {
      result.push(
        arr[i][0].toUpperCase() + arr[i].slice(1, arr[i].length).toLowerCase()
      );
    } else {
      result.push(arr[i]);
    }
  }
  return result.join(" ");
}

File: 프로그래머스/2/12951. JadenCase 문자열 만들기/JadenCase 문자열 만들기.java
class Solution {
    public String solution(String s) {
        String answer = "";
        String[] splitArr = s.split(" ");

        for (String string : splitArr) {

            if (string.isEmpty()) {
                answer += " ";
            } else{
                answer += string.substring(0, 1).toUpperCase() + string.substring(1).toLowerCase() + " ";
            }
        }

        if(s.charAt(s.length() - 1) == ' '){
            return answer;
        }

        return answer.substring(0, answer.length()-1);
    }
}